lib/api/model/model.dart

```diff
@@ -664,6 +665,37 @@ enum MessageFlag {
   String toJson() => _$MessageFlagEnumMap[this]!;
 }
 
+@JsonSerializable(fieldRename: FieldRename.snake)
+class MessageEditHistory {
+
+  final String? prevContent;
+  final String? prevRenderedContent;
+  final int? prevRenderedContentVersion;
+  final int? prevStream;
+  final String? prevTopic;
+  final int? stream;
+  final int timestamp;
+  final String? topic;
+  final int? userId;
+
+  MessageEditHistory( {
+    required this.prevContent,
+    required this.prevRenderedContent,
+    required this.prevRenderedContentVersion,
+    required this.prevStream,
+    required this.prevTopic,
+    required this.stream,
+    required this.timestamp,
+    required this.topic,
+    required this.userId,
+  });
+
+  factory MessageEditHistory.fromJson(Map<String, dynamic> json) =>
+      _$MessageEditHistoryFromJson(json);
```

Comment:
nit: two-space indents
```suggestion
  factory MessageEditHistory.fromJson(Map<String, dynamic> json) =>
    _$MessageEditHistoryFromJson(json);
```